packages/client/src/actions/listToken.ts

```diff
@@ -48,8 +48,7 @@ export async function listToken(
         !('fee' in listing) &&
```

Comment:
I think we also need to change the check here because each listing should now supply just a fee (`feeRecipient` was removed from the api). I think this will also need to be change in RK ui package's ListModal, to pass in the correct data.